code/controllers/subsystem/SSticker.dm

```diff
@@ -341,93 +339,45 @@ SUBSYSTEM_DEF(ticker)
 				var/turf/T = get_turf(M)
-				if(T && is_station_level(T.z) && !istype(M.loc, /obj/structure/closet/secure_closet/freezer) && !(issilicon(M) && override == "AI malfunction"))
-					to_chat(M, "<span class='danger'><B>The blast wave from the explosion tears you atom from atom!</B></span>")
-					var/mob/ghost = M.ghostize()
+				if(T && is_station_level(T.z) && !istype(M.loc, /obj/structure/closet/secure_closet/freezer))
+					M.ghostize()
 					M.dust() //no mercy
-					if(ghost && ghost.client) //Play the victims an uninterrupted cinematic.
-						ghost.client.screen += cinematic
 					CHECK_TICK
-			if(M && M.client) //Play the survivors a cinematic.
-				M.client.screen += cinematic
 
 	//Now animate the cinematic
 	switch(station_missed)
-		if(1)	//nuke was nearby but (mostly) missed
+		//nuke was nearby but (mostly) missed
+		if(1)
 			if(mode && !override)
 				override = mode.name
 			switch(override)
 				if("nuclear emergency") //Nuke wasn't on station when it blew up
-					flick("intro_nuke", cinematic)
-					sleep(35)
-					SEND_SOUND(world, sound('sound/effects/explosion_distant.ogg'))
-					flick("station_intact_fade_red", cinematic)
-					cinematic.icon_state = "summary_nukefail"
+					play_cinematic(/datum/cinematic/nuke/ops_miss, world)
 				if("fake") //The round isn't over, we're just freaking people out for fun
-					flick("intro_nuke", cinematic)
-					sleep(35)
-					SEND_SOUND(world, sound('sound/items/bikehorn.ogg'))
-					flick("summary_selfdes", cinematic)
+					play_cinematic(/datum/cinematic/nuke/fake, world)
 				else
-					flick("intro_nuke", cinematic)
-					sleep(35)
-					SEND_SOUND(world, sound('sound/effects/explosion_distant.ogg'))
-
-
-		if(2)	//nuke was nowhere nearby	//TODO: a really distant explosion animation
-			sleep(50)
-			SEND_SOUND(world, sound('sound/effects/explosion_distant.ogg'))
+					play_cinematic(/datum/cinematic/nuke/self_destruct_miss, world)
+		//nuke was nowhere nearby	//TODO: a really distant explosion animation
+		if(2)
+			play_cinematic(/datum/cinematic/nuke/far_explosion, world)
 		else	//station was destroyed
 			if(mode && !override)
 				override = mode.name
 			switch(override)
 				if("nuclear emergency") //Nuke Ops successfully bombed the station
-					flick("intro_nuke", cinematic)
-					sleep(35)
-					flick("station_explode_fade_red", cinematic)
-					SEND_SOUND(world, sound('sound/effects/explosion_distant.ogg'))
-					cinematic.icon_state = "summary_nukewin"
+					play_cinematic(/datum/cinematic/nuke/ops_victory, world)
 				if("AI malfunction") //Malf (screen,explosion,summary)
-					flick("intro_malf", cinematic)
-					sleep(76)
-					flick("station_explode_fade_red", cinematic)
-					SEND_SOUND(world, sound('sound/effects/explosion_distant.ogg'))
-					cinematic.icon_state = "summary_malf"
+					play_cinematic(/datum/cinematic/malf, world)
+				if("blob") //Station nuked (nuke,explosion,summary)
```

Comment:
Is there a reason to single out blob here when it's the same as the else case? Also, if this would only fire in the blob gamemode, you can remove this condition entirely since we don't even have it in code anymore